src/azure-cli/HISTORY.rst

```diff
@@ -38,6 +38,7 @@ Release History
 * vm create: Use standard public IP SKU automatically when using zones.
 * vm create: Compose a valid computer name from VM name if computer name is not provided.
 * vm create: Add --workspace to enable log analytics workspace automatically.
+* vm create: Do not allow specifying accelerated networking when using an existing NIC.
```

Comment:
Well. It must be a breaking change. Right? Should we deprecated it first?

Reply:
Kind of. Can't use --accelerated-networking and --nics togother now. But no one is deprecated. What's the workflow of this condition?

Reply:
I think I can use warning first. Then after several releases, change it to error.

Reply:
Resolved. Added a reminder to onenote to change it to error later.